cdist/conf/type/__link/explorer/type

```diff
@@ -31,6 +31,7 @@ elif [ -f "$destination" ]; then
    type="$(cat "$__object/parameter/type")"
    case "$type" in
       hard)
+         # TODO SC2012: use stat?
          link_count=$(ls -l "$destination" | awk '{ print $2 }')
```

Comment:
`stat --format '%h' "$destination"` is perfect on Linux -- FreeBSD according to <https://www.freebsd.org/cgi/man.cgi?query=stat&apropos=0&sektion=0&manpath=FreeBSD+11.2-RELEASE+and+Ports&arch=default&format=html> `stat -f '%l' "$destination`